homeassistant/components/assist_pipeline/select.py

```diff
@@ -51,7 +71,7 @@ class AssistPipelineSelect(SelectEntity, restore_state.RestoreEntity):
     _attr_options = [OPTION_PREFERRED]
 
     def __init__(self, hass: HomeAssistant, unique_id_prefix: str) -> None:
-        """Initialize a pipeline selector."""
+        """Initialize a vad sensitivity selector."""
```

Comment:
This doc string still belongs to the pipeline selector.
```suggestion
        """Initialize a pipeline selector."""
```